src/config.ts

```diff
@@ -6,8 +6,8 @@ export const config = () => {
     version: '0.1',
     /** Configuration related with session */
     session: {
-      /** TimeOut in Minutes default 24 hours */
-      sessionTimeOut: 24 * UNIT_TIME.HOUR * MILLISECONDS,
+      /** TimeOut in Minutes default 10 hours */
+      sessionTimeOut: 10 * UNIT_TIME.HOUR * MILLISECONDS,
       /** Extended Session in Minutes default 30 minutes */
```

Comment:
Why we move this to 10 hours? I discussed with @lucasponce about and we saw that openshift has 24 hours for this.